docs/jade-getting-started.md

```diff
@@ -292,14 +322,36 @@ to build `jade-data-repo`.
 * Certain environment variables need to be set beforehand. Instances of `zzz`
 should be replaced by your initials or the environment (i.e. `dev`):
 
+
 ```
+export NVM_DIR="$HOME/.nvm"
+export JAVA_HOME=$(/usr/libexec/java_home)
```

Comment:
Now that we are front loading these variables, this list seems pretty overwhelming imo. A number of these variables seem unnecessary. I think we should only include the absolute minimum required to run integration tests.

Reply:
Good point - I think I've now narrowed it down to just the important ones. I added a note about converting between application.properties and environment variables, which should cover any settings I left out.